src/tokens/functional/color/light/patterns-light.json5

```diff
@@ -1931,6 +1931,12 @@
       },
     },
   },
+  skeletonLoader: {
```

Comment:
I didn't add color tokens for the spinner because we just use `currentColor` with opacity. In most cases, this will mean the "track" of the spinner is `fgColor.default` @ 25% opacity, the bit that spins around the track is `fgColor.default` @ 100% opacity.